test/unit/lodash.spec.js

```diff
@@ -0,0 +1,568 @@
+        it('should return converted string', function () {
+                expect(_.escape.apply(_, params)).to.eql('Tom &amp; Jerry are old friends');
+            }
+        );
+    });
+
+    describe('escapeRegExp', function () {
+        before(function () {
+            params = datasets.escapeRegExp();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should return converted string', function () {
+                expect(_.escapeRegExp.apply(_, params)).to.eql('\\[realskill\\]\\(https://realskill\\.com/\\)');
+            }
+        );
+    });
+
+    describe('kebabCase', function () {
+        before(function () {
+            params = datasets.kebabCase();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('shold return transformed string', function () {
+                expect(_.kebabCase.apply(_, params)).to.eql('some-string-to-test-function');
+            }
+        );
+    });
+
+    describe('lowerCase', function () {
+        before(function () {
+            params = datasets.lowerCase();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should return string converted to lowercase', function () {
+                expect(_.lowerCase.apply(_, params)).to.eql('some string to test function');
+            }
+        );
+    });
+
+    describe('lowerFirst', function () {
+        before(function () {
+            params = datasets.lowerFirst();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should return converted string with first letter in lowercase', function () {
+                expect(_.lowerFirst.apply(_, params)).to.eql('sOME STRING TO TEST FUNCTION');
+            }
+        );
+    });
+
+    describe('pad', function () {
+        before(function () {
+            params = datasets.pad();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(Number.isInteger(params[1])).to.eql(true);
+            expect(params[1]).to.be.above(params[0].length);
+            expect(params).to.have.length.within(2, 3);
+            if (params[2] !== undefined && params.length === 3) {
+                expect(params[2]).to.be.a('string');
+            }
+        });
+        it('should return converted string with padding characters', function () {
+                expect(_.pad.apply(_, params)).to.eql('._._.some long sweet text._._.');
```

Comment:
In my opinion, without looking to this file, that solution is impossible to reveal